specification/cluster/allocation_explain/examples/request/ClusterAllocationExplainRequestExample2.yaml

```
@@ -0,0 +1,53 @@
+summary: Query Parameters
+method_request: GET _cluster/allocation/explain
```

this is incorrect, all the parameters need to be added to the method_request field for the automation to generate the examples correctly
```suggestion
method_request: GET _cluster/allocation/explain?index=my-index-000001&shard=0&primary=false&current_node=my-node
```